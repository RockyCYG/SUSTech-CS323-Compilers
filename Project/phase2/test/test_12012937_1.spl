int test(char t){
	int a = t + t;
	return t+t;
}
struct A{
};
struct B{
	int b;
};

int main()
{
	struct A aa;
	struct A bb;
	int a[10][10];
	int b[15];
	int x=1.0;
	//a=b;
	int tt=aa+bb;
	{
		//int y=5;
		x=y+x;
	}
	return 1;
	return 0;
}	

