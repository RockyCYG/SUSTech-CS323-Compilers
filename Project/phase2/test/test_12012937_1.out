Error type 7 at Line 2: binary operation on non-number variables
Error type 5 at Line 2: unmatching type on both sides of assignment
Error type 7 at Line 3: binary operation on non-number variables
Error type 8 at Line 3: incompatiable return type
Error type 5 at Line 17: unmatching type on both sides of assignment
Error type 7 at Line 19: binary operation on non-number variables
Error type 5 at Line 19: unmatching type on both sides of assignment
Error type 1 at Line 22: undefined variable: y
Error type 7 at Line 22: binary operation on non-number variables

